.github/workflows/check-docs.yml

```diff
@@ -1,30 +0,0 @@
-name: Check docs build
-# This workflow runs when a PR is labeled with `docs`
-# This will check if the docs build successfully by running `npm run build`
-on:
```

Comment:
do we run this check somewhere else? This is building in pr phase and the other in push to main. We can merge the two but we should still check build. We had a few occurrences where things were merged but would cause the build for docs to fail

Reply:
Alright, will add!